addons/survey/tests/test_certification_flow.py

```diff
@@ -150,6 +150,26 @@ def test_flow_certification(self):
         self.assertEqual(certification_email.attachment_ids[0].name, f'Certification - {certification.title}.html',
                          'Default certification report print_report_name is "Certification - %s" % (object.survey_id.display_name)')
 
+        # Check that the certification can be printed without access to the participant's company
+        with self.with_user('admin'):
+            new_company = self.env['res.company'].create({
+                'name': 'newB',
+            })
+            npu = self.env['res.users'].create({
+                'name': 'No print right user',
+                'login': 'npu',
+                'password': 'npu',
+                'groups_id': [(6, 0, [
+                        self.ref('base.group_user'),
+                        self.ref('survey.group_survey_user')
+                    ])],
+                'company_id': new_company.id,
+                'company_ids': [new_company.id],
+            })
+            new_company.invalidate_model() # cache pollution
+            self.env['ir.actions.report'].with_user(npu).with_company(new_company)\
+                    ._render_qweb_pdf('survey.certification_report_view', res_ids=user_inputs.ids)[0]
```

Comment:
With this and green build, LGTM.
```suggestion
                ._render_qweb_pdf('survey.certification_report_view', res_ids=user_inputs.ids)
```